### ISSUE TITLE
为什么finetune speech_seaco_paraformer_large_asr_nat-zh-cn-16k-common-vocab8404-pytorch，验证集Update best acc: 0.0000

### ISSUE THREAD

**Issue by juzstu** [modelscope/FunASR#2012]
## 🐛 Bug

finetune speech_seaco_paraformer_large_asr_nat-zh-cn-16k-common-vocab8404-pytorch，在保存模型的时候，每个 epoch 的Update best acc: 0.0000

### Environment

 - OS (Linux):
 - FunASR Version (1.1.5):
 - ModelScope Version (1.17.1):
 - PyTorch Version (2.2.1):
 - How you installed funasr (`pip`,):
 - Python version: 3.8
 - GPU (V100M32)
 - CUDA/cuDNN version (cuda11.7):

 [2024-08-14 12:23:13,004][root][INFO] - train, rank: 0, epoch: 2/100, data_slice: 0/1, step_in_slice: 150/1, step_in_epoch: 150, total step: 786, (loss_avg_rank: 0.057), (loss_avg_slice: 0.132), (ppl_avg_slice: 1.141e+00), (acc_avg_slice: 0.000), (lr: 1.315e-05), [('loss_att', 0.555), ('acc_att', 0.727), ('loss_seaco', 0.051), ('loss', 0.057)], {'data_load': '0.001', 'forward_time': '0.500', 'backward_time': '0.593', 'optim_time': '0.039', 'total_time': '1.138'}, GPU, memory: usage: 2.886 GB, peak: 25.514 GB, cache: 26.412 GB, cache_peak: 31.336 GB
[2024-08-14 12:25:47,060][root][INFO] - train, rank: 0, epoch: 2/100, data_slice: 0/1, step_in_slice: 200/1, step_in_epoch: 200, total step: 836, (loss_avg_rank: 0.031), (loss_avg_slice: 0.133), (ppl_avg_slice: 1.142e+00), (acc_avg_slice: 0.000), (lr: 1.398e-05), [('loss_att', 0.206), ('acc_att', 0.167), ('loss_seaco', 0.029), ('loss', 0.031)], {'data_load': '0.001', 'forward_time': '1.151', 'backward_time': '1.879', 'optim_time': '0.065', 'total_time': '3.100'}, GPU, memory: usage: 4.069 GB, peak: 25.514 GB, cache: 26.434 GB, cache_peak: 31.336 GB
[2024-08-14 12:25:47,062][root][INFO] - Validate epoch: 2, rank: 0
[2024-08-14 12:26:37,690][root][INFO] - val, rank: 0, epoch: 2/100, data_slice: 0/1, step_in_slice: 50/1, step_in_epoch: 50, total step: 50, (loss_avg_rank: 0.089), (loss_avg_slice: 0.071), (ppl_avg_slice: 1.073e+00), (acc_avg_slice: 0.000), (lr: 0.000e+00), [('loss_att', 0.389), ('acc_att', 0.853), ('loss_seaco', 0.085), ('loss', 0.089)], {'data_load': '0.000', 'forward_time': '1.228', 'total_time': '1.229'}, GPU, memory: usage: 5.149 GB, peak: 25.514 GB, cache: 26.434 GB, cache_peak: 31.336 GB
[2024-08-14 12:27:07,175][root][INFO] - Save checkpoint: 2, rank: 0, local_rank: 0
[2024-08-14 12:27:17,628][root][INFO] -
Checkpoint saved to /xxxl/model.pt.ep2.200
[2024-08-14 12:27:37,626][root][INFO] - Update best acc: 0.0000, /xxx/model.pt.best

**Comment by HengYuh** [comment]
你好，我也遇到了这个问题，我推测可能是因为输入的数据只有source和target导致其他推理为0的原理，请问你解决了这个问题嘛
